h2olog/.gitattributes

```diff
@@ -0,0 +1 @@
+generated-*.cc linguist-generated=true
```

Comment:
How about moving this file to root?

Reply:
Done! b489a8e4b793c22c2fc67abcdc87e22cebf9b818